airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/embedder.py

```diff
@@ -173,8 +173,8 @@ def check(self) -> Optional[str]:
             return format_exception(e)
         return None
 
-    def embed_chunks(self, chunks: List[Chunk]) -> List[List[float]]:
-        return self.embeddings.embed_documents([chunk.page_content for chunk in chunks])
+    def embed_chunks(self, chunks: List[Chunk]) -> List[Optional[List[float]]]:
+        return cast(List[Optional[List[float]]], self.embeddings.embed_documents([chunk.page_content or "" for chunk in chunks]))
```

Comment:
at this point, the page content can't even be empty, it's purely for mypy... I guess this should be refactored a bit. the content is optional now because we have the "do not store raw text" flag now, but for embedding this doesn't matter. there probably should be a separate type for that. I will pull out this refactoring in a separate PR.